src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -397,10 +455,27 @@ public synchronized Color getPixelColor(int x, int y) {
     /**
      * Creates an image containing pixels read from the screen.  This image does
      * not include the mouse cursor.
```

Comment:
We have seen in macOS specifically that absence of mouse cursor is not guaranteed and screen capture can contain mouse cursor, so I guess we should remove this line..

Reply:
Sure, this is a good time to do it.